frontend/javascripts/oxalis/view/right-border-tabs/bounding_box_tab.js

```diff
@@ -120,6 +106,14 @@ function BoundingBoxTab(props: BoundingBoxTabProps) {
           />
         </Tooltip>
       </div>
+      {selectedBoundingBoxForExport != null ? (
+        <ExportBoundingBoxModal
+          dataset={dataset}
+          tracing={tracing}
+          boundingBox={selectedBoundingBoxForExport.boundingBox}
+          destroy={() => setSelectedBoundingBoxForExport(null)}
```

Comment:
I think, `destroy` can be renamed to `onClose` or something similar. The `destroy` naming is usually only used in the context of `renderIndependently` (maybe we should avoid that function in the future to avoid similar bugs which are related to store access).